unit-tests/heitech.zer0mqXt.core.tests/heitech.zer0mqXt.core.tests.csproj

```diff
@@ -15,6 +15,7 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="NSubstitute" Version="4.0.0"/>
+    <PackageReference Include="FluentAssertions" Version="6.5.1"/>
```

Comment:
I like analyzers - should we also include the [FluentAssertions.Analyzers](https://www.nuget.org/packages/FluentAssertions.Analyzers/) package?

E.g. that analyzer helped me to replace
`collection.Count.Should().Be(61)`
to
`collection.Should().HaveCount(61)`
where to me then the assertion failed message is slightly better - and I learned a new feature about `FluentAssertions`.

Reply:
Sweet, I just set up[ a new issue ](https://github.com/TimoHeiten/zer0mqXt/issues/62) -  if it is fine with you?

Reply:
Absolutely. I might actually grab that one 😉.